v2/cmd/wails/internal/commands/initialise/templates/templates/svelte/frontend/README.md

```diff
@@ -1,105 +1,48 @@
-*Looking for a shareable component template? Go here --> [sveltejs/component-template](https://github.com/sveltejs/component-template)*
+# Svelte + Vite
 
----
+This template should help get you started developing with Svelte in Vite.
 
-# svelte app
+## Recommended IDE Setup
 
-This is a project template for [Svelte](https://svelte.dev) apps. It lives at https://github.com/sveltejs/template.
+[VSCode](https://code.visualstudio.com/) + [Svelte](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode).
```

Comment:
```suggestion
  - [Goland](https://www.jetbrains.com/go/) + [Svelte](https://plugins.jetbrains.com/plugin/12375-svelte)
  - [VSCode](https://code.visualstudio.com/) + [Svelte](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode).
```

Reply:
Sorry, I forgot it, I just overwritten it, do I need to go back to the previous version now?